VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
@anurse Why to remove `*.opensdf`? It is still used (for old database engine in VS 2015). And temporary file with `*.VC.VC.opendb` extension (yes, two subsequent `VC`) is used when solution is opened. I think that all of them must stay in the gitignore for VS.